mesh/mesh.cpp

```diff
@@ -5160,6 +5163,19 @@ void Mesh::Loader(std::istream &input, int generate_edges,
                   "invalid mesh: end of file tag not found");
    }
 
+   if (NURBSext && NURBSext->NonconformingPatches())
```

Comment:
AStyle failed to format this section properly. I'm not sure why -- maybe it gets confused parsing this file.